pkg/controller/managedresource/merger.go

```diff
@@ -30,13 +30,16 @@ const (
 	descriptionAnnotationText = `DO NOT EDIT - This resource is managed by gardener-resource-manager.
 Any modifications are discarded and the resource is returned to the original state.`
 
-	originAnnotation = "resources.gardener.cloud/origin"
+	originAnnotation       = "resources.gardener.cloud/origin"
+	annotationKeyReplicas  = "gardener-resource-manager.gardener.cloud/preserveReplicas"
+	annotationKeyResources = "gardener-resource-manager.gardener.cloud/preserveResources"
```

Comment:
Please move the two new constants to https://github.com/gardener/gardener-resource-manager/blob/4d38c292feea2ad3d653f824ea659fb81bded192/api/resources/v1alpha1/types.go#L22 as they are part of the "API contract".

Reply:
Ok sure, thank you